libpod/runtime_ctr.go

```diff
@@ -343,9 +343,6 @@ func (r *Runtime) setupContainer(ctx context.Context, ctr *Container) (_ *Contai
 					if err != nil {
 						return nil, errors.Wrapf(err, "error retrieving pod %s cgroup", pod.ID())
 					}
-					if podCgroup == "" {
```

Comment:
possibly a warn here so user knows pod cgroup is not set? is this handled elsewhere?

Reply:
my mistake, this change doesn't belong to this PR.  I've dropped it.

Thanks for noticing it!